src/libs/tal-chorus/Chorus.h

```diff
@@ -108,6 +105,8 @@ class Chorus
 			*writePtr= 0.0f;
 		}
 		while (++writePtr < delayLineEnd);
+
+		writePtr = delayLineStart;
```

Comment:
Actual fix ☝🏻

Reply:
If we re-assign `writePtr = delayLineStart` after the prior while-loop incrementing, then the final incremented `writePtr` value is overwritten.

Seems like we can drop the while-loop?

Reply:
Hmm, we need it for zeroing out the buffer.

Reply:
Ahh sorry - the "do { .. }" body was collapsed in GitHub :sweat_smile:  (All I was seeing was an empty while loop; makes sense now!)